File: FutsalSystem/FutsalSystem/Models/DTO/Player/PlayerDTO.cs
using System;
using FutsalSystem.Base.Interface;

namespace FutsalSystem.Models.DTO.Player
{
    public class PlayerDTO : IBaseEntity
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime DateOfBirth { get; set; }
        public int YellowCardsCount { get; set; }
        public int RedCardsCount { get; set; }
        public int MatchesPlayed { get; set; }
        public string ImagePath { get; set; }
        public int Goals { get; set; }
        public int OwnGoals { get; set; }
        public int? Number { get; set; }
        public int? TeamId { get; set; } // komandos id, kurioje zaidzia
        public string TeamName { get; set; }
    }
}

File: FutsalSystemFront/src/app/Models/PlayerDto.ts
export class PlayerDto {
    public Id: number;
    public FirstName: string;
    public LastName: string;
    public DateOfBirth: Date;
    public ImagePath: string;
    public YellowCardsCount: number;
    public RedCardsCount: number;
    public MatchesPlayed: number;
    public Goals: number;
    public OwnGoals: number;
    public Number: number;
    public TeamId: number;
    public TeamName: string;

    constructor(id: number, firstName: string, lastName: string, dateOfBirth: Date, imagePath: string, yellowCardsCount: number, redCardsCount: number, matchesPlayed: number, goals: number, ownGoals: number, number: number, teamId: number, teamName: string) {
        this.Id = id;
        this.FirstName = firstName;
        this.LastName = lastName;
        this.DateOfBirth = dateOfBirth;
        this.ImagePath = imagePath;
        this.YellowCardsCount = yellowCardsCount;
        this.RedCardsCount = redCardsCount;
        this.MatchesPlayed = matchesPlayed;
        this.Goals = goals;
        this.OwnGoals = ownGoals;
        this.Number = number;
        this.TeamId = teamId;
        this.TeamName = teamName;
    }
}